Services/UserService/Validators/CreateUserRequestValidator.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DTO;
+using DTO.BrokerRequests;
+using DTO.RestRequests;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>
+    {
+        public CreateUserRequestValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .WithMessage(ErrorsMessages.EmailEmpty)
+                .EmailAddress()
+                .WithMessage(ErrorsMessages.InvalidEmail);
+
+            RuleFor(user => user.Password)
```

Comment:
Aren't you check passwordhash here?  It will have different handling

Reply:
Because passwordhash generate after my validation